src/Tempest/Console/src/Commands/Generators/MakeControllerCommand.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Commands\Generators;
+
+use Tempest\Console\ConsoleArgument;
+use Tempest\Console\ConsoleCommand;
+use Tempest\Console\Stubs\ControllerStub;
+use Tempest\Generation\HasGeneratorCommand;
+
+final class MakeControllerCommand
+{
+    use HasGeneratorCommand;
+
+    #[ConsoleCommand(
+        name       : 'make:controller',
+        description: 'Create a new controller class with a basic route',
+        aliases    : ['controller:make', 'controller:create', 'create:controller'],
+    )]
+    public function __invoke(
+        #[ConsoleArgument(
+            help: 'The name of the controller class to create ( "Controller" will be suffixed )',
+        )]
+        string $className,
+        #[ConsoleArgument(
+            help: 'The route path inside the controller',
+        )]
+        ?string $controllerPath = null,
+        #[ConsoleArgument(
+            help: 'The view name to return in the controller',
+        )]
+        ?string $controllerView = null,
+    ): void {
+        $suggestedPath = $this->getSuggestedPath(
+            className  : $className,
+            pathPrefix : 'Controllers',
+            classSuffix: 'Controller',
```

Comment:
```suggestion
            className: $className,
            pathPrefix: 'Controllers',
            classSuffix: 'Controller',
```

I am not sure if specifying a path prefix and a class suffix is a good idea. This enforces conventions that not everyone would want to follow.

For instance, I personally don't put controllers in a `Controllers` directory. But right now, this is hardcoded, and will always be suggested to the user when running `make:controller`.

This is why, in my initial implementation, I wanted to simply publish the file to the root of the main namespace, so there is no specific opinion about it. This is one of the core features of Tempest: everything can be wherever, there's no convention.

Reply:
That's true, I was seeing that as an extra work, but in case I guess we could keep those options on the helpers because it can be useful.
But I'll remove it from the internal commands 👍